src/components/CommentCard.jsx

```diff
@@ -1,8 +1,37 @@
 import Card from 'react-bootstrap/Card';
 import Col from 'react-bootstrap/Col';
 import ThumbUpRoundedIcon from '@mui/icons-material/ThumbUpRounded';
+import DeleteForeverIcon from '@mui/icons-material/DeleteForever';
+import { deleteComment } from './utils/api';
+import { useState } from 'react';
+import Toast from 'react-bootstrap/Toast';
 
-function CommentCard({comment}) {
+
+function CommentCard({comment, setComments, user}) {
+  const [err, setErr] = useState(null);
+
+  function removeComment(){
+    setErr(null);
+    setComments((currentComments) => {
+      return currentComments.filter((item) => {
+        if (comment.comment_id === item.comment_id){
+          return false;
+        }
+
+        return true;
+      })
+    })
+
+    deleteComment(comment.comment_id)
+    .catch((error) => {
+      setComments((currentComments) => {
+        const updatedComments = [...currentComments, comment];
+        return updatedComments;
+      })
+      setErr("Something went wrong, please try again")
+    })
+
```

Comment:
Works perfectly when there's internet and the comment is persistently deleted 👍 
However, when there is an error with the delete request (for example when the internet drops) it still does appear to that the comment is deleted, it only reappears when you either refresh the page or navigate away and back again
It seems like this catch block isn't working as intended, perhaps because there's another catch block in the deleteComment function?